reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -120,7 +121,8 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
             ),
             repository = input.ortResult.repository,
             repositoryConfiguration = yamlMapper.writeValueAsString(input.ortResult.repository.config),
-            labels = input.ortResult.labels
+            labels = input.ortResult.labels,
+            metaData = MetaDataCalculator().getMetaData(input.ortResult)
```

Comment:
The `MetaDataCalculator` class seems so small - how about inlining it below line 101?

Reply:
We had this discussion ourselves, but as we expect that more data will be added to this model, we thought that it is beneficial to add a separate `Calculator` right now instead of refactoring at a later point.